packages/react-native-bpk-component-chip/src/__snapshots__/BpkChip-test.android.js.snap

```diff
@@ -284,7 +284,7 @@ exports[`Android BpkChip dark mode should render outline correctly 1`] = `
         },
         Object {
           "backgroundColor": "transparent",
-          "borderColor": "rgb(68, 69, 96)",
+          "borderColor": "rgb(72, 72, 74)",
```

Comment:
why have these changed?

Reply:
I'm guessing due to colour changes from the tokens, as didn't you update the black tints, in `bpk-tokens`?

Reply:
Looks like it might be this colour changed: `colorBlackTint05`

Reply:
Shouldn't have changed from `29.6.1` to `30.1.0` though 🤔 
https://github.com/Skyscanner/backpack/blob/master/CHANGELOG.md#2020-01-21---renamed-elevation-tokens-for-android

Reply:
I believe lineColor has changed

Reply:
Ah yeah - it'll be line colour 👍